src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.Shared/Harness/VisualStudioInstanceFactory.cs

```diff
@@ -408,6 +408,7 @@ private static async Task<Process> StartNewVisualStudioProcessAsync(string insta
             IntegrationHelper.KillProcess("dexplore");
 
             var process = Process.Start(vsExeFile, vsLaunchArgs);
+            TakeSnapshotEveryTimeSpanUntilProcessExit(process, $"devenv{process.Id}");
```

Comment:
Oh, I take an extra look so here this function would wait for the process until it exits.
Is this something you expecting?
I feel the real thing we want is to have a seperate thread take snapshots periodically. And listen to the devenv Process exit event, is that correct?

Reply:
OK it should be fixed now!